### PR TITLE
LRDOCS-3803 - Included required identical credentials when remote sta…

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown [hunk 1]
```diff
@@ -278,7 +280,15 @@ Settings* &rarr; *Foundation*.
...
-You should proceed only when all three major steps have been completed. 
+4. The same users must exist on both remote and the staging environments. In
+addition, the users must have the same credentials (that is, same screen name,
+email, and roles, password).  
+
+**Important note:** do not simply clone the database or else the Remote
+Publication process will also fail.
+
+
+You should proceed only when all four major steps have been completed.
```
Comment (sez11a) [inline discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown]: @codyhoag, can you verify this information?

Comment (codyhoag) [inline discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown]: Yeah, I'll verify this with Máté. He may also have some additional info related to this too. I'll let you know.

**Comment by justinchoi001** [comment]
@sez11a , did you close the LRDOCS ticket? The PR is still open.

**Comment by sez11a** [comment]
@justinchoi001 Yup; by mistake. It's reopened.

**Comment by codyhoag** [comment]
@sez11a Added a couple more details for this PR in sez11a#2403.

File: discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown (hunk 1 above)
Comment (sez11a) [inline discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown]: @codyhoag Did you ever get any information about this?

Comment (codyhoag) [inline discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown]: @sez11a I commented at the bottom of this PR. I resent with additional details in sez11a#2403.

Comment (sez11a) [inline discover/portal/articles/02-web-experience-management/03-staging-content-for-publication/01-enabling-staging.markdown]: @codyhoag  Oops. Thank you.

**Comment by sez11a** [comment]
Superseded by https://github.com/sez11a/liferay-docs/pull/2403

**Pull request closed without merging**